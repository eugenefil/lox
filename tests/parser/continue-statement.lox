while true { continue; }
for ch in "foo" { continue; }
