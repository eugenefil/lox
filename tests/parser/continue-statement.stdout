(program
  (while
    true
    (block
      (continue)))
  (for
    ch
    "foo"
    (block
      (continue))))
